### PR TITLE
Post to tweet about new task when it was approved.

### PR DESCRIPTION
**Pull request by thepry**
SHOULD NOT be merged until sidekiq jobs are fixed (see #44).

This PR relates to #54 "Post approved tasks in twitter account" and adds:
- New sidekiq worker for twetting about new tasks
- Creates a sidekiq job after task was approved
- Adds env variables required by twitter gem to .env files.

Also I haven't been able to test actual posting, 'cause twitter requires to attach phone number before you can register app, and somehow I don't receive twitter sms messages :/

Keys required by gem:
```
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_ACCESS_TOKEN=
TWITTER_ACCESS_SECRET=
```

### CODE REVIEW

File: apps/admin/controllers/moderation/update.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def call(params)
       if params[:action] == 'approve'
         TaskRepository.new.update(params[:id], { approved: true })
         ApproveTaskWorker.perform_async(params[:id])
+        ApprovedTaskTwitterWorker.perform_async(params[:id])
```
Comment (davydovanton) [inline apps/admin/controllers/moderation/update.rb]: can we use service instead twitter worker and put this service to `ApproveTaskWorker`?

File: lib/ossboard/workers/approved_task_twitter_worker.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class ApprovedTaskTwitterWorker
+  include Sidekiq::Worker
+
+  def perform(task_id)
+    task = TaskRepository.new.find(task_id)
+    return unless task
+    tweet "#{task.title} #ossboard #{link_to_task(task_id)}"
```
Comment (davydovanton) [inline lib/ossboard/workers/approved_task_twitter_worker.rb]: and what we will do if task title will more than 140 chars?

File: apps/admin/controllers/moderation/update.rb (hunk 1 above)
Comment (thepry) [inline apps/admin/controllers/moderation/update.rb]: Yep!

File: lib/ossboard/workers/approved_task_twitter_worker.rb (hunk 2 above)
Comment (thepry) [inline lib/ossboard/workers/approved_task_twitter_worker.rb]: Good catch.
I'll limit it

**Comment by thepry** [comment]
@davydovanton Done! You can merge it when sidekiq is fixed

**Pull request merged**